src/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -226,6 +226,12 @@ export function getUiSettingDefaults() {
       value: '($0,0.[00])',
       description: 'Default <a href="http://numeraljs.com/" target="_blank">numeral format</a> for the "currency" format'
     },
+    'format:language': {
+      value: 'en',
+      type: 'select',
+      options: ['be-nl', 'chs', 'cs', 'da-dk', 'de-ch', 'de', 'en', 'en-gb', 'es-ES', 'es', 'et', 'fi', 'fr-CA', 'fr-ch', 'fr', 'hu', 'it', 'ja', 'nl-nl', 'pl', 'pt-br', 'pt-pt', 'ru-UA', 'ru', 'sk', 'th', 'tr', 'uk-UA'],
+      description: '<a href="http://numeraljs.com/" target="_blank">numeral language</a>'
```

Comment:
Could you please add [`rel="noopener"`](https://mathiasbynens.github.io/rel-noopener/) to this link.